src/python/m5/stats/gem5stats.py

```diff
@@ -230,7 +230,7 @@ def _prepare_stats(group: _m5.stats.Group):
     for stat in group.getStats():
         stat.prepare()
 
-    for child in getStatGroups().values():
+    for child in group.getStatGroups().values():
```

Comment:
Just be careful here. This has already been fixed: https://gem5-review.googlesource.com/c/public/gem5/+/42704 (this will be merged into develop this afternoon: https://gem5-review.googlesource.com/c/public/gem5/+/43265).

Reply:
Hm. How to deal with this then? Wait until this is merged with develop and then rebase or something?

Reply:
Exactly :)

Reply:
@powerjg Doesn't look like this change is reflected in the github repo, am I supposed to cherry-pick this commit from googlesource or something like that?